### DISCUSSION TITLE
Which matchers-condition are available?

### DISCUSSION THREAD

**Discussion by lollomart** [projectdiscovery/nuclei#836] [Q&A] [answered]
Hi guys, I am new on nuclei.

Anyone can tell me please which matcher-condition are available?
I can't find this information in the docs / Q&A /others.
I know that there are these two one: OR and AND.

I would like to do something like: "if there is not a specified path in a http response , than show me that in the specified site is missing that path.".
I can't find a way to do this and I don't know if it is actually possibile.

Thank you so much for the attention :)

**Comment by ehsandeep** [comment, marked as answer]
@lollomart check this out - https://nuclei.projectdiscovery.io/templating-guide/

**Reply by lollomart** [reply]
@ehsandeep Thank you!
My fault, I didn't saw the part regarding matchers on the site.
I've been trying to look for it on the http raw request section.

Problem solved, thanks!
